ArduCopter/control_autotune.cpp

```diff
@@ -260,6 +273,126 @@ bool Copter::autotune_start(bool ignore_checks)
     return true;
 }
 
+const char *Copter::autotune_level_issue_string() const
+{
+    switch (autotune_level_problem.issue) {
+    case Copter::AUTOTUNE_LEVEL_ISSUE_NONE:
+        return "None";
+    case Copter::AUTOTUNE_LEVEL_ISSUE_ANGLE_ROLL:
+        return "Angle(R)";
+    case Copter::AUTOTUNE_LEVEL_ISSUE_ANGLE_PITCH:
+        return "Angle(P)";
+    case Copter::AUTOTUNE_LEVEL_ISSUE_ANGLE_YAW:
+        return "Angle(Y)";
+    case Copter::AUTOTUNE_LEVEL_ISSUE_RATE_ROLL:
+        return "Rate(R)";
+    case Copter::AUTOTUNE_LEVEL_ISSUE_RATE_PITCH:
+        return "Rate(P)";
+    case Copter::AUTOTUNE_LEVEL_ISSUE_RATE_YAW:
+        return "Rate(Y)";
+    }
+    return "Bug";
```

Comment:
Multiple, "Bug" is defined. If the same pointer is OK. If it is different, I think that it is better to make it a common variable.